### PR TITLE
Pisa weight fitter

### PR DESCRIPTION
**Pull request by RasmusOrsoe**
adds the ability to use pisa to fit physical weights to each events in a database

### CODE REVIEW

File: src/graphnet/pisa/fitting.py
Comment (asogaard) [inline src/graphnet/pisa/fitting.py:18]: Due to refactor in #215 
```suggestion
from graphnet.data.sqlite import run_sql_code, save_to_sql
```

File: src/graphnet/pisa/fitting.py
Comment (asogaard) [inline src/graphnet/pisa/fitting.py:79]: A docstring here, or in `fit_weights`, would be helpful to explain which weights are being fitted, exactly.

File: src/graphnet/pisa/fitting.py
Comment (asogaard) [inline src/graphnet/pisa/fitting.py:81]: Here and elsewhere we are assuming that databases (as opposed to some other intermediate file format) are being used to interface with PISA. Is that something we are committing to, or just something that works for the time being but might be expanded on down the line?

**Review by asogaard — COMMENTED** [review]
Hi Rasmus,
I don't have a good handle on the PISA portion of the repo, so I am happy to merge this PR without much back-and-forth. I have made one suggestion though, to update an import that was changed on a recent PR, that I suggest we add before merging. I have also suggested adding just a bit of documentation to explain the intent behind the `WeightFitter` class, as this is not super clear to me at present, but I'll leave that up to you. :)

**Review by asogaard — APPROVED** [review]

File: src/graphnet/pisa/fitting.py
Comment (RasmusOrsoe) [reply src/graphnet/pisa/fitting.py:81]: This is something we can expand upon down the line, I think. Currently, GraphNeT is only interfaced with PISA via SQL. To make it work with a new format, we need to update PISA as it relies on a dataloader for each format. See https://github.com/icecube/pisa/tree/master/pisa/stages/data for reference. They are easy to write, so it shouldn't be a big thing.

**Pull request merged**
